src/ui/Views/Organisation/Courses.cshtml

```diff
@@ -0,0 +1,91 @@
+@model CourseListViewModel;
+
+@{
+  ViewBag.Title = Model.ProviderName + " courses";
+}
+
+@if (Model.MultipleOrganisations) {
```

Comment:
There should be a breadcrumb if it's single org, eg the same breadcrumb as this minus the first crumb.